controllers/admin/AdminStoresController.php

```diff
@@ -333,7 +333,7 @@ public function postProcess()
             $langs = Language::getLanguages(false);
             /* Cleaning fields */
             foreach ($_POST as $kp => $vp) {
-                if (!in_array($kp, ['checkBoxShopGroupAsso_store', 'checkBoxShopAsso_store', 'hours'])) {
+                if (is_string($kp)) {
```

Comment:
```suggestion
                if (is_string($vp)) {
```

Reply:
@PululuK Oooooh you're right 😮  I did not see that.

One good reason to avoid short variable names like this 😅